Dockerfile

```
@@ -1,103 +1,108 @@
-# Create a user and set up sudo access
-RUN groupadd -r appuser && useradd -m -r -g appuser appuser && \
-    mkdir -p /etc/sudoers.d && \
-    echo "appuser ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/appuser && \
-    chmod 440 /etc/sudoers.d/appuser
-COPY --chown=appuser:appuser AIVEN_SSL.pem /app/AIVEN_SSL.pem
-COPY . /app/
-# Switch to the user
-USER appuser
 
+# Copy Doppler CLI install script
+RUN (curl -sLf --retry 3 --tlsv1.2 --proto "=https" https://packages.doppler.com/public/cli/install.sh) | sh -s -- --no-install-cli-version-file --install-path /install/bin
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)